iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/BatteryImpl.java

```diff
@@ -17,6 +17,8 @@
  */
 public class BatteryImpl extends AbstractConnectable<Battery> implements Battery, ReactiveLimitsOwner {
 
+    private final NetworkImpl network;
```

Comment:
Use a Ref instead and get the network in the getter of setter, otherwise merging is broken.